### PR TITLE
Collected small changes for the next release

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This pull request collects multiple small changes and fixes.

**Related Issue(s)**

N/A

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

N/A

**Implementation Notes**

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [x] The added/updated documentation is integrated and tested with the documentation build system
- [x] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

### CODE REVIEW

File: src/REAXFF/fix_acks2_reaxff.cpp [hunk 1]
```diff
@@ -94,8 +94,8 @@ FixACKS2ReaxFF::~FixACKS2ReaxFF()
   memory->destroy(s_hist_X);
   memory->destroy(s_hist_last);
 
-  deallocate_storage();
-  deallocate_matrix();
+  FixACKS2ReaxFF::deallocate_storage();
```
Comment (stanmoore1) [inline src/REAXFF/fix_acks2_reaxff.cpp:97]: Why is this necessary?

Comment (akohlmey) [reply src/REAXFF/fix_acks2_reaxff.cpp:97]: Within the destructor you don't have access to the virtual function tables.
Adding the scope makes this more obvious. Functionality is unchanged.

**Review by stanmoore1 — APPROVED** [review]
I approve

**Review by sjplimp — APPROVED** [review]
